.github/workflows/mac-mps.yml

```diff
@@ -33,6 +33,18 @@ jobs:
       MACOS_SCCACHE_S3_ACCESS_KEY_ID: ${{ secrets.MACOS_SCCACHE_S3_ACCESS_KEY_ID }}
       MACOS_SCCACHE_S3_SECRET_ACCESS_KEY: ${{ secrets.MACOS_SCCACHE_S3_SECRET_ACCESS_KEY }}
 
+  macos-12-py3-arm64-test:
```

Comment:
I'm a bit confuse looking at this.  Does this make `mac-mps` workflow look exactly like `trunk` when it comes to MacOS testing?  In that case, would that be easier to have an auto label rule to add `ciflow/trunk` for every mps-related PR instead?

Reply:
I think the idea is that all of trunk takes too long and people working on mps tend to merge as soon as mps is finished

I'm honestly not too sure how necessary this is given that it's only happened once, so if people have thoughts on whether this should happen, I'm open to it

Reply:
The tradeoff here sounds like either making this mac-mps workflow a bit more complex (with some code duplication) vs running trunk more often on mac workflows (causing more jobs to be run and potentially some job duplication)

Given the focus on capacity constraints and not wanting to use more infrastructure than needed, I'd lean towards adding an extra job in this mac-mps workflow than adding ciflow/trunk to all mps builds